string movement
int32 duration
---
string movement2
